### PR TITLE
Fixes #33502 - align host owner in host details

### PR DESCRIPTION
**Pull request by amirfefer**
Screen Shot 2021-10-03 at 18 48 46

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
Do we want the paded line to move?
Looks weird to split other attributes from Host owner by the line

**Comment by amirfefer** [comment]
How about moving the host group as well?  something like:
Screen Shot 2021-10-04 at 17 20 47

This also addresses [this issue](https://projects.theforeman.org/issues/33501)

**Comment by amirfefer** [comment]
@MariSvirik /cc

**Comment by ezr-ondrej** [comment]
> How about moving the host group as well?

Ack on that! I'm ok to merge if we move Host group as well, that would make sense to me :+1:

**Comment by MariSvirik** [comment]
@amirfefer Hey, I'm fine with that. 
Just fix sentence case, please: 'Host owner', 'Host group', 'Mac address', 'IPv4 address', 'iPv6 address'

**Comment by amirfefer** [comment]
@ezr-ondrej , @MariSvirik 
 done :)

File: webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/styles.scss [hunk 1]
```diff
@@ -6,3 +6,12 @@
...
 }
+
+.truncate {
+  text-overflow: ellipsis;
+  max-width: 90%;
+  white-space: nowrap;
+  overflow: hidden;
+  display: inline-block;
+  margin-right: 5px;
+}
```
Comment (ekohl) [inline webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/styles.scss]: Please ensure the newline at the end.

**Review by ekohl — COMMENTED** [review]
Do you happen to have a screenshot with the new layout?

File: webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/styles.scss (hunk 1 above)
Comment (amirfefer) [reply webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/styles.scss]: Thanks, I forgot to clean this

**Comment by amirfefer** [comment]
@ekohl, sure:
Screen Shot 2021-10-05 at 16 45 03

**Review by ekohl — APPROVED** [review]
Looks correct to me. I still think IPv4 and IPv6 addresses deserve the same. Remember that an IPv6 address can be `2a02:16a8:dc:b0d:7e19:17fc:284:5ef0` or even longer (dc can be 4 characters, some for the other parts). However, the current issue doesn't describe that so I'm fine with doing it later.

**Review by ezr-ondrej — APPROVED** [review]
Thanks @amirfefer ! :+1:

**Comment by ezr-ondrej** [comment]
CP 3.0: #8842

**Pull request merged**